docker/nugus_sim/run.py

```diff
@@ -23,13 +23,21 @@
 
 
 def read_config(file_path):
-    with open(file_path, "r") as file:
-        return yaml.load(file)
+    try:
+        with open(file_path, "r") as file:
+            return yaml.load(file)
+    except FileNotFoundError:
+        print(f"config file not found: {file_path}")
+        sys.exit(1)
```

Comment:
How about returning an empty dictionary (maybe also ensuring directories exist)? Just in case the config doesn't already exist, this would allow you to create it

Reply:
I reckon we should fail early with the error if we're attempting to load a config that's not in the image. Creating the config could lead to unexpected behaviour if we later read it in the code and try to access a key that doesn't exist.